src/utils/ready.js

```diff
@@ -4,6 +4,9 @@ export default function ready(context, fn, req) {
     return fn(context.webpackStats);
   }
 
-  context.log.info(`wait until bundle finished: ${req.url || fn.name}`);
+  const name = req.url || fn.name;
+
+  context.log.info(`wait until bundle finished${name ? `: ${name}` : ''}`);
+
```

Comment:
before - when you run `invalidate` method it output: `wait until bundle finished: `
now - `wait until bundle finished`